pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -301,6 +305,50 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 	return nil
 }
 
+// Activate moves the given pods to activeQ iff they're in unschedulableQ or backoffQ.
+func (p *PriorityQueue) Activate(pods map[string]*v1.Pod) {
+	p.lock.Lock()
+	defer p.lock.Unlock()
+
+	activated := false
+	for _, pod := range pods {
+		if p.activate(pod) {
+			activated = true
```

Comment:
We should `break` here?

Or all/one of pods activate, we call `p.cond.Broadcast()`?